src/dos/program_mount.cpp

```diff
@@ -59,7 +59,8 @@ void MOUNT::Move_Z(char new_z)
 		/* Update environment */
 		std::string line = "";
 		std::string tempenv = {new_drive_z, ':', '\\'};
-		if (first_shell->GetEnvStr("PATH",line)) {
+		if (auto result{first_shell->GetEnvStr("PATH")}) {
```

Comment:
That syntax is a bit too funky... just to get rid of a temporary in non performance intensive code that the compiler will optimise away anyway?

Can we use standard non-surprising syntax for regular use cases?